--- Qt5/init-repository           2017-11-18 01:08:01.840841229 +0000
+++ Qt5/init-repository           2017-11-18 01:39:30.226422713 +0000
@@ -468,8 +468,8 @@

     my $do_clone = (! -e "$submodule/.git");
     if ($do_clone) {
-        push @reference_args, '--branch', $branch;
+        push @reference_args, '--no-checkout';
         $self->exe('git', 'clone', @reference_args,
                    ($mirror ? $mirror : $url), $submodule);
     }
